packages/core/src/editor/BlockNoteEditor.ts

```diff
@@ -282,7 +282,10 @@ export class BlockNoteEditor<
   private onUploadStartCallbacks: ((blockId?: string) => void)[] = [];
   private onUploadEndCallbacks: ((blockId?: string) => void)[] = [];
 
-  public readonly resolveFileUrl: (url: string) => Promise<string>;
+  public readonly resolveFileUrl: (
+    url: string,
+    blockId?: string
```

Comment:
shouldn't this parameter be required? Or are there cases when it's undefined? If so, I think we should document those